src/commands/api-mesh/init.js

```diff
@@ -86,7 +93,7 @@ class InitCommand extends Command {
 		let shouldCreateGit = gitFlagOptions[flags.git];
 		let packageManagerChoice = flags.packageManager;
 		const packageJsonTemplate = `${getAppRootDir()}/src/templates/package.json`;
-		const dotNpmrcPath = `${getAppRootDir()}/src/templates/.npmrc`;
```

Comment:
why change from .npmrc to npmrc?

Reply:
While publishing `.` files are not published. For instance, here is the latest published package on npm, as you can see none of the `.` files are published

<img width="877" alt="image" src="https://github.com/adobe/aio-cli-plugin-api-mesh/assets/35203638/a9ab1ed7-fff6-4953-ac54-adcb72e1d253">

Similarly, `template/.npmrc` is also missing which wont let the init command to complete the setup

<img width="890" alt="image" src="https://github.com/adobe/aio-cli-plugin-api-mesh/assets/35203638/49154105-6268-459d-9ecb-7c05f72adc73">

Hence changing it to `npmrc` instead.

Reply:
This is from the npm docs, it says `.npmrc` along with a bunch of other files is always ignored https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files

<img width="768" alt="image" src="https://github.com/adobe/aio-cli-plugin-api-mesh/assets/35203638/17a5469d-dc2a-4607-b699-62df050b5e30">

Reply:
ahh. This requires a .npmignore file and you can then specify the files you want in it like so
{
  "files": ["index.js", "/lib"]
}
This whitelists everything inside and you can test it out by running npm pack

Reply:
But `.npmrc` is always ignored irrespective of `files` array or `.npmignore`